### PR TITLE
Remove docker_local from Makefile

### PR DESCRIPTION
**Pull request by kirtanchandak**
- removed `docker_local` from makefile as instructed from the discussion -
 https://github.com/vitessio/website/pull/1777

### CODE REVIEW

**Review by frouioui — APPROVED** [review]
FYI, linked to https://github.com/vitessio/website/pull/1781

Looks good to me! Happy first contribution to vitess!

File: Makefile [hunk 1]
```diff
@@ -331,9 +331,6 @@ docker_lite_all: docker_lite $(DOCKER_LITE_TARGETS)
 docker_lite:
 	${call build_docker_image,docker/lite/Dockerfile,vitess/lite}
 
-docker_local:
-	${call build_docker_image,docker/local/Dockerfile,vitess/local}
-
 docker_run_local:
 	./docker/local/run.sh
```
Comment (frouioui) [inline Makefile:335]: Let's also remove this please, we do not need it anymore.

Comment (frouioui) [reply Makefile:335]: this = `docker_run_local`

Comment (kirtanchandak) [reply Makefile]: removed this too!

**Review by deepthi — APPROVED** [review]
Should be merged after https://github.com/vitessio/website/pull/1781

**Pull request merged**